packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -577,11 +551,11 @@ const DatePickerInput = (props) => {
       return;
     }
     // decrement current valid date by 1 day, if not valid set date to yesterday instead
-    if (event.key === '-' || event.key === '_') {
+    if (event.key === '-' || event.key === '_' || event.keyCode === KEY_DOWN) {
       if (validDate) {
-        inputDate = DateUtil.decrementDateByDay(inputDate, DateUtil.ISO_EXTENDED_DATE_FORMAT);
+        inputDate = DateUtil.decrementDateByDay(inputDate, DateUtil.ISO_EXTENDED_DATE_FORMAT, inputTypeValue);
       } else {
-        inputDate = DateUtil.decrementDateByDay(DateUtil.getCurrentDate(), DateUtil.ISO_EXTENDED_DATE_FORMAT);
+        inputDate = DateUtil.decrementDateByDay(DateUtil.getCurrentDate(), DateUtil.ISO_EXTENDED_DATE_FORMAT, inputTypeValue);
       }
```

Comment:
Let's rename this function to `decrementDate`

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1999/commits/7b61de3934e318f97325758b279207800dc84d9e